### DISCUSSION TITLE
Why is Prisma generating "WHERE 1=1" with every request?

### DISCUSSION THREAD

**Discussion by ArtoszBart** [prisma/prisma#25550] [Q&A] [answered]
### Question

I have logged queries that Prisma generates, and I noticed that to every request prisma adds "WHERE 1=1". Why is that? Is there any way to disable it?

For example:
`await prisma.issue.findMany();`

generated query:
```
SELECT "public"."Issue"."id", "public"."Issue"."title", "public"."Issue"."description", "public"."Issue"."status"::text, "public"."Issue"."createdAt", "public"."Issue"."updatedAt", "public"."Issue"."assignedToUserId" 
FROM "public"."Issue" 
WHERE 1=1 
OFFSET $1
```

**Comment by nurul3101** [comment, marked as answer]
Hello @ArtoszBart 👋 

Can you try adding a filtering condition in your `findMany` query and check if you still get `WHERE 1=1` in the resulting query?
I believe you are getting `WHERE 1=1` because your findMany query has no parameters/no where conditions.

Prisma adds "WHERE 1=1" as a placeholder when no actual filters are applied to the query. This is done to maintain a consistent query structure, especially when dynamic conditions might be added later. It's essentially a no-op condition that always evaluates to true and doesn't affect the query results.

**Reply by nurul3101** [reply]
@ArtoszBart Checking to see if you had a chance to look at my last message. Let me know in case you have any queries. Feel free to Mark the Answer and Close this discussion if your query is resolved.
